src/views/Cart/scss/index.scss

```diff
@@ -76,20 +76,50 @@
   &__empty {
     text-align: center;
     padding: $spacer * 5 0;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    height: 100%;
+    position: relative;
+    justify-content: center;
 
     h4 {
       font-weight: $bold-font-weight;
-      text-transform: uppercase;
       margin-bottom: $spacer;
     }
 
     p {
-      color: $gray;
+      color: $dark-gray;
     }
 
     &__action {
       text-align: center;
       margin-top: $spacer;
     }
+    &-icon {
+      background-size: cover;
+      background-repeat: no-repeat;
+      width: 262px;
+      height: 190px;
+      margin-bottom: 45px;
```

Comment:
Use `$spacer` or `rem` units for margin, paddings.

Will just comment this one, but fix the others too.

Reply:
Sorry didn't your last statement